src/averageAges.js

```diff
@@ -20,6 +20,23 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let men;
+
+  century ? (men = people.filter(person => {
+    return person.sex === 'm' && century === Math.ceil(person.died / 100);
+  })) : (men = people.filter(person => {
+    return person.sex === 'm';
+  }));
+
+  const menAges = men.map(oneMan => oneMan.died - oneMan.born);
+
+  const totalAverageAge = menAges.reduce((sum, currentAge) => {
+    return sum + currentAge;
+  }, 0);
+
+  const averageAge = totalAverageAge / menAges.length;
+
```

Comment:
move it to the separate function so you will be able ti reuse it in your main functions